build.gradle

```diff
@@ -82,11 +70,9 @@ jar {
 		'Class-Path': configurations.shadow.files.stream()
 				.filter($it -> !$it.name.startsWith('javafx')).collect{"lib/$it.name"}.join(' ')
 	)
-	exclude 'licenses/'
 	from 'LICENSE'
 	dependsOn minifyCss
 	dependsOn copyRuntimeLibs
```

Comment:
`copyRuntimeLibs` needs to be adjusted too, because it still references `configurations.shadow`.

Those libs are needed for the *-min.jar as well as for bundling with the Windows Installer.